Marlin/Marlin_main.cpp

```diff
@@ -6524,30 +6518,25 @@ void Stop() {
   }
 }
 
-bool setTargetedHotend(int code){
+bool setTargetedHotend(int code) {
   target_extruder = active_extruder;
   if (code_seen('T')) {
     target_extruder = code_value_short();
     if (target_extruder >= EXTRUDERS) {
-      SERIAL_ECHO_START;
-      switch(code){
+      switch(code) {
```

Comment:
...and frankly, for concision, the `switch` isn't really needed anymore either. :-) This function obviously doesn't need to police the caller and add one more place to itemize weird un-named numeric codes.